wandb/sdk/wandb_run.py

```diff
@@ -408,7 +408,7 @@ def _init(
         self._step = 0
         self._torch_history: Optional["wandb.wandb_torch.TorchHistory"] = None
 
-        self._start_time = time.time()
+        self._start_time = self._settings._start_time or time.time()
```

Comment:
I think this one is on purpose (not sure) cc: @raubitsj

Reply:
ah, right. I think I remember someone even complaining about it. @raubitsj should I revert it?

Reply:
I think andrew wanted it for importing a run but it doesnt feel like the best way to support this... lets revert this for now unless you tested all the downstream parts of this... if you set start time to 2 days ago and run a test for 15 minutes.. does the total time get calculated right?  my guess without looking at the code fully is that the runtime might be 2days and 15 minutes.

Reply:
ah, yea, it was Andrew, right! Got it, let me revert for now, it's not related to this PR (but will add a todo).